modules/packages/LinearAlgebra.chpl

```diff
@@ -806,14 +806,16 @@ proc inv (ref A: [?Adom] ?eltType, overwrite=false) where usingLAPACK {
 }
 
 /*
-  Return the matrix ``A`` to the ``bth`` power, where ``b`` is a positive
-  integral type.
+  Return the matrix ``A`` to the ``bth`` power, where ``b`` is an integer.
 
   .. note::
 
     ``matPow`` will utilize the :mod:`BLAS` module for improved performance, if
     available. Compile with ``--set blasImpl=none`` to opt out of the
     :mod:`BLAS` implementation.
+
+    If ``b`` is negative, ``matPow`` will depend on :mod:`LAPACK` module, and will
```

Comment:
How about putting this in a new note, and formatting similarly to previous note for easier reading?

e.g.

```
.. note::

    ``matPow`` will depend on the :mod:`LAPACK` module when ``b`` is negative. 
    A runtime error will occur if ``b<0`` and ``lapackImpl`` is ``none``.